src/app/features/location/address-edit/address-edit.component.ts

```diff
@@ -114,8 +116,12 @@ export class AddressEditComponent {
       geoLookup: value?.geoLookup,
     });
   }
-}
 
-function matchGeoResults(a: GeoResult, b: GeoResult) {
-  return a.lat === b.lat && a.lon === b.lon;
```

Comment:
@sleidig I removed this unused function as part of a cleanup. I double-checked, and it doesn't appear to be used anywhere in the code.